### ISSUE TITLE
Render issue if only one x-axis label

### ISSUE THREAD

**Issue by mpskovvang** [chartist-js/chartist#372]
If you only provide one label the graph won't render.

Try this snippet:

``` javascript
new Chartist.Line('.ct-chart', {
  labels: ['Monday'],
  series: [
    [12]
  ]
}, {
  fullWidth: true,
  chartPadding: {
    right: 40
  }
});
```

But if you add 'Tuesday' to the labels it will work.

**Comment by gionkunz** [comment]
We need to figure out how to display 1 data point. Probably just the point on line charts. Let's keep this  issue in mind.

**Comment by gionkunz** [comment]
This is fixed with #800 and 1b32910
